simple/index.mjs

```diff
@@ -0,0 +1,158 @@
+ import 'dotenv/config';
+
+import express from 'express';
+import mysqlx from '@mysql/xdevapi';
+
+import config from './config.mjs';
+
+const app = express();
+
+app.set('view engine', 'pug');
+
+const port = process.env.PORT || 7777;
```

Comment:
This port should probably be held in a constants file

Reply:
I agree, magic numbers are typically frowned upon